### ISSUE TITLE
Advanced Move help text

### ISSUE THREAD

**Issue by chrsmthws** [brian-frederick/Discord-MonsterBot#5]
> ADVANCEMOVE
> Allows for advanced outcome of a move.
> - Key (text) (required)
> The key of the move to advance as part of a hunter improvement. Such as ksa, aup, ho, iam, ms, ps, rabs, um or the key for a special move.
> - Tag (mention)
> @ another hunter to use their properties instead of the sender's.
> AKA: advance, adv

ADVANCEMOVE
AKA abbreviations: advance, adv
.m advancemove [key]
.m advance [key]
.m adv [key]

- Key (text, required)
Keys are represented by their abbreviations, such as ksa, aup, ho, iam, ms, ps, rabs, um, or the key for a special/custom move. (e.g. "Claws of the Beast" has been given the key "cotb")
In this case, it will be the move the hunter is advancing.

This will update your hunter's stats to offer the corresponding outcome text of an advanced move.
In gameplay, if you're hunter has advanced a basic move and makes a +12 roll, you will receive the appropriate outcome text.

Example: .m advance ksa

![Advance KSA](https://user-images.githubusercontent.com/74613905/99430164-98fca400-28ce-11eb-94ba-a5c324cc0170.png)

Now, if your hunter gets +12 on a .m ksa, you and the Keeper will see the consequence options.

![Advance KSA Roll](https://user-images.githubusercontent.com/74613905/99430197-a6199300-28ce-11eb-993c-617e017e0567.png)

If you'd like to update another hunter's stats (i.e. roll for them), tag the hunter like so:

Example: .m ************* [I don't know how to do this]

**Comment by brian-frederick** [comment]
There's some good expansions of the basic command help template here. I'm not sure I agree with all of it but I like a lot of it. 

For instance, it seems unnecessary to provide examples of usage with each alias.

".m advancemove [key]
.m advance [key]
.m adv [key]"

BUT it does seem worth showing the permutations of usage for each different parameter, as in:

".m advancemove [key]
.m advancemove [key] [tag]"

I like the idea of examples but it seems super labor intensive just to collect screenshots, etc.

And your description of the command is definitely better so I'll swap that out. Feel free to rewrite any of the existing descriptions and add them here in a comment. I think we can use this thread to overhaul how we do command help.

**Comment by chrsmthws** [comment]
Yeah I'm just coming at it from as dumb a place as I already am, pretending I have little to no idea how any of this works or awareness of vocabulary that might be common.
